dojo/tools/sslyze/parser.py

```diff
@@ -6,14 +6,14 @@ class SslyzeParser(object):
     """SSLYze support JSON and XML"""
 
     def get_scan_types(self):
-        return ["SSLyze 3 Scan (JSON)", "Sslyze Scan"]
+        return ["Sslyze Scan", "SSLYze Scan(JSON)"]
```

Comment:
As you are changing the scan_type value, we will need a DB migration.

Reply:
I am afraid to say I caused the problem initially, when I set the scan type to "SSLyze 3 Scan (JSON)". Now we have SSLyze5. I think we have to change the scan type. Tomorrow I will do a test, what's the best way to do it.

Reply:
Yes, we will need a DB migration if we change the scan type, otherwise reimports are not possible. While testing the reimport I saw a problem with the deduplication settings I want to investigate. 

@kibernautas Do you mind if I do these 2 issues in your branch?

Reply:
Not at all be my guest :D